advanced/Scripts/piholeDebug.sh

```diff
@@ -712,50 +662,6 @@ check_networking() {
     check_required_ports
 }
 
-check_x_headers() {
```

Comment:
Are we ditching the idea of a blockpage entirely with the NGweb?

Reply:
As of now, there's no plans that I know of to add a block page. It is feasible however; the API can support a server-side rendered page. The reason for server-side rather than client-side is that we don't want to alarm the user's browser by running JavaScript on a blocked (possibly malicious) domain.

Reply:
I'm just thinking that there are still going to be users that want that functionality, even though we recommend otherwise.

Server side rendered page is probably OK though? Work out all the values in the API and then just send raw HTML back to the browser. I like that idea!

Reply:
Should this be added to the MVP requirements? @pi-hole/core-approvers

Reply:
A team discussion decided to make this a requirement for release, but not necessarily for alpha/beta release. See pi-hole/api#130